command_test.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2016 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package updater
+
+import (
+	"os"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/keybase/go-logging"
+)
+
+var log = logging.Logger{Module: "test"}
+
+func TestEmptyRunCommand(t *testing.T) {
+	out, err := RunCommand("", nil, time.Second, log)
```

Comment:
Ok, got the tests down to 30ms and not testing with a fake clock.. I think that's preferable.